docs/core/cwc-badge.md

```diff
@@ -12,10 +12,10 @@ import '@clr/core/badge';
 
 ## Properties
 
-| Property | Attribute | Type                                             | Default      | Description                                      |
-|----------|-----------|--------------------------------------------------|--------------|--------------------------------------------------|
-| `color`  | `color`   | `"gray" \| "purple" \| "blue" \| "orange" \| "light-blue"` | **required** | Sets the color of the badge from a predefined list of choices |
-| `status` | `status`  | `"info" \| "success" \| "warning" \| "danger"`   | **required** | Sets the color of the badge from a predefined list of statuses |
+| Property | Attribute | Type     | Default      | Description                                      |
+|----------|-----------|----------|--------------|--------------------------------------------------|
+| `color`  | `color`   | `string` | **required** | Sets the color of the badge from a predefined list of choices <br/> (`gray`, `purple`, `blue`, `orange`, `light-blue`) |
```

Comment:
I don't think these two files were supposed to be generated. The attr comments were supposed to be deleted but I think got added back in the merge. Since they are now props that get reflected back as attrs we won't need the `@attr` syntax. This also improves the types, listing the exact options vs only `string`.

https://github.com/vmware/clarity/commit/22e4a7264c22159fa0f96d3b450db26b7e116c9a#diff-54331ce7c79dc0e1dde04d68564fa734L26

https://github.com/vmware/clarity/commit/22e4a7264c22159fa0f96d3b450db26b7e116c9a#diff-880cda4012a8b2bd5e267508a624da89L39

Reply:
Ah yeah let me fix this up